no-computer.md

```diff
@@ -1,69 +1,175 @@
+   * [Regroup](http://www.sumopaint.com/images/temp/xzakkncxrfefpjbf.png) -
+     wait for the group and do an activity with everyone
+   * [Turkey Eagle](http://www.sumopaint.com/images/temp/xzrrtdejhdtnhjhs.png)
+     used to indicate an easy and hard path, hounds each choose one.
+     Followed by the paths eventually joining.
+   * [BVC](http://www.sumopaint.com/images/temp/xzkritmbkfjnhjpc.png) - warns
+     hounds to be very careful on stairs, in streets, etc.
+   * [ON-IN](http://www.sumopaint.com/images/temp/xzrhifdistbcmeit.png) -
+     indicates the end of the route.
+
+Activities can also refer to a number of known concepts for specifying who
+should do an activity:
+   * FRB[xN] - 'Front Running B------' (or first N people).
+   * DFL[xN] - 'Dead F----- Last' (or last N people).
+   * N - the number of people on the hash.
+
+HL routes thus consist of chains of these symbols linked by arrows. Programs
+can be written well (arrows line up cleanly) or contain errors (arrows do not
+line up).
 
 ### Operation (semantics)
-_What might happen when a program runs? How does a program interact with the
-user? What kinds of errors might occur, and how might they be communicated to
-the user?_
 
+An HL route is executed by being followed by the hounds. In this sense it is
+interpreted rather than compiled - no holistic check of the route is done
+before it begins executed.
+
+The hounds follow the route to the best of their ability, much the same as a
+computer follows assembly instructions. Many types of errors can occur.
+
+HL routes can be malformed - for example lines instead of arrows, or not
+re-uniting the turkey and eagle paths. They can also be well-formed, but
+produce undesired behavior - for example having a loop of arrows :wink: or
+early arrows may lead the hounds to late arrows, skipping the middle section
+of the route altogether.
+
+### Why is it a language?
+
+HL is a language for three reasons.
+
+It is a standardized method of communication, governed by rules and unambiguous
+when used correctly.
+
+It is written by one entity (the hares) and thus used by that entity to
+the behavior of another entity (the hounds).
+
+It has fundamental units which allow for composition into arbitrarily complex
+routes. In this way novel routes can be written forever.
 
 ### Expressiveness
-_What should be easy to do in this language? What should be possible, but
-difficult? What should be impossible or very difficult?_
 
+Leading people along routes is very easy in HL. One might imagine getting a
+computer to follow such a route would be very difficult.
+
+HL also makes it difficult to lay routes in some areas, including:
+
+   * Indoors
+   * Over slippery surfaces where chalk doesn't stick
+   * Over grass
+   * Over gravel
+   * In water, although sometimes it works out well (the CMC cube fountain)
+
+These may not be desired difficulties, but nevertheless they exist because of
+the constraints of chalk.
```

Comment:
Have there ever been talk to use other mediums either instead of or in addition to?